src/objects/oo/zcl_abapgit_oo_base.clas.abap

```diff
@@ -231,4 +249,10 @@ CLASS ZCL_ABAPGIT_OO_BASE IMPLEMENTATION.
     DELETE FROM seocompotx WHERE clsname = is_key-clsname. "#EC CI_SUBRC
     INSERT seocompotx FROM TABLE it_descriptions.         "#EC CI_SUBRC
   ENDMETHOD.
+
+
+  METHOD zif_abapgit_oo_object_fnc~update_descriptions_sub.
+    DELETE FROM seosubcotx WHERE clsname = is_key-clsname. "#EC CI_SUBRC
+    INSERT seosubcotx FROM TABLE it_descriptions.         "#EC CI_SUBRC
```

Comment:
can we do something here to make sure CLSNAME is the correct class?

Reply:
we could compare to the `clsname` in `it_description` but that won't work if there are no descriptions at all. 

or we refactor the `zcl_abapgit_oo_base` to set the class name in the constructor and use it instead of import parameters:

https://github.com/abapGit/abapGit/blob/0be78203d50e513bfaab4c3cd536b6ff63eb34f0/src/objects/zcl_abapgit_object_clas.clas.abap#L106

Reply:
just loop through IT_DESCRIPTIONS and set CLSNAME = is_key-clsname ?

Reply:
I removed the class/interface name for shorttexts from the XML